packages/gatsby-plugin-catch-links/src/catch-links.js

```diff
@@ -146,12 +146,12 @@ export const routeThroughBrowserOrApp = (
 
   if (hashShouldBeFollowed(origin, destination)) return true
 
-  if (pluginOptions.excludeRegex) {
+  if (pluginOptions.excludePattern) {
```

Comment:
Let's make sure we support both cases, just in case.

```suggestion
  if (pluginOptions.excludeRegex && !pluginOptions.excludePattern) {
    pluginOptions.excludePattern = pluginOptions.excludeRegex
  }

  if (pluginOptions.excludePattern) {
```